### ISSUE TITLE
Readonly to_parquet and glue table version support

### ISSUE THREAD

**Issue by samcon** [awslabs/aws-data-wrangler#353]
Hey,
First of all - awswrangler is great!
I'm currently using wr.s3.to_parquet to convert JSON documents to Parquet using a schema I defined in a glue table and write them to S3. 
The table also has partition projection enabled and since the JSON to Parquet conversion is done with the glue table's schema (and no dynamically) - I'm not interested in modifying the glue table in any way.
I might have missed it, but It would be great to have a switch that disables any writing to/modifying of the glue table. 

On a similar subject, it'd be super useful to allow specifying the exact glue table version to use. 
i.e. in catalog.py:478 use get_table_version instead of get_table (and of course allow passing version_id from to_parquet)

Thanks!

**Comment by igorborgest** [comment]
Thanks @samcon! Your ideas are welcome!

1 - You are right, currently it is not possible to disable table update at all for `wr.s3.to_parquet()`. What behaviour do you want if you receive a different schema? Raise an exception? Ignore the write?

2 - About `wr.s3.to_parquet()` in old table versions, I'm not sure if I got the point. After the write do you want to create a new version cloning the old one pointed through the arguments? Or do you want to keep the used version in the same position?

**Comment by samcon** [comment]
Thanks for the quick answer Igor!

1. Good question. I guess if there's a mismatch in the schema I'd like for the write not to go through + get an exception.

2. It's just that are situations where you'd like to lock the schema (in this case, use a specific glue table version) that you're using
Consider the following: 
Writer1 performs to_parquet on some data using schema_v1
schema_v1 is changed out of band in schema_v2 is created
Writer2 performs to_parquet on some data using schema_v1
Writer1, that is still running, should still be able to convert using schema_v1
Does that make sense?

**Comment by igorborgest** [comment]
1. Looks perfect for me, let's tag it for our next minor release `1.9.0`!

2. I see your point and I will run some tests about it, let's try to figure out the best way to address that.

**Comment by samcon** [comment]
Awesome! Many thanks for the quick response and LMK if I can be of any assistance!

**Comment by igorborgest** [comment]
Hi @samcon 

I've just added the `schema_evolution` boolean argument in our `dev` branch through the commit above. I think it will be useful for you as `schema_evolution=False`.

Do you mind to test it from our development branch and check if it helps in your use case?
The ideia is to publish this enhancement in the version `1.9.0` later this week.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released on version **[1.9.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.9.0)**
